EOLib/Net/PacketBuilder.cs

```diff
@@ -67,7 +67,12 @@ public IPacketBuilder AddChar(byte b)
 
         public IPacketBuilder AddShort(short s)
         {
-            return AddBytes(_encoderService.EncodeNumber((ushort)s, 2));
+            return AddBytes(_encoderService.EncodeNumber(s, 2));
+        }
+
+        public IPacketBuilder AddShort(ushort s)
```

Comment:
See my comment on #149 - could you elaborate on the behavior this fixes? I wasn't seeing issues with shorts > 32767 as session IDs because the cast to `ushort` ensured the correct value was encoded.

I would prefer not to add this overload if it isn't needed

Reply:
I’ll test it again. It seemed like it was overflowing or something. I kept getting a really huge number back on reoserv when the session id was bigger than a signed short can hold

Reply:
You're right casting to (ushort) before encoding totally fixes it. Not sure why the code didn't have that when I was testing.. Maybe I deleted it?? lmao

Works now without the overload :)